app/(gcforms)/[locale]/(form administration)/form-builder/Start.tsx

```diff
@@ -42,21 +43,21 @@ export const Start = () => {
       fileReader.onload = (e) => {
         if (!e.target || !e.target.result || typeof e.target.result !== "string") return;
         const data = safeJSONParse(e.target.result, cleaner);
```

Comment:
We should type `data` at the source.


```suggestion
        const data = safeJSONParse<FormRecord> (e.target.result, cleaner);
```